src/legacy/core_plugins/timelion/public/shim/legacy_dependencies_plugin.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import chrome from 'ui/chrome';
+import { Plugin } from 'kibana/public';
+// @ts-ignore
+import { VisFactoryProvider } from 'ui/vis/vis_factory';
+import { npStart, npSetup } from 'ui/new_platform';
+import { initTimelionLegacyModule } from './timelion_legacy_module';
+
+/** @internal */
+export interface LegacyDependenciesPluginSetup {
+  $http: any;
+  config: any;
+  createAngularVisualization: Function;
+}
+
+/** @internal */
+export interface LegacyDependenciesPluginStart {
+  $rootScope: any;
```

Comment:
not sure if we need this. AngularVisController will inject this at runtime (we don't need them at setup time)

Reply:
How would be better to replace the dependency in `...timechart/schema.ts`?